.buildkite/pipelines/pull_request/base.yml

```diff
@@ -110,8 +111,43 @@ steps:
         - exit_status: '*'
           limit: 1
     artifact_paths:
-      - "target/kibana-security-serverless/**/*"
+      - "target/kibana-security-solution/**/*"
+    
+  - command: .buildkite/scripts/steps/functional/security_serverless_investigations.sh
+    label: 'Serverless Security Investigations Cypress Tests'
```

Comment:
cc @banderror we'll have to remember to add this for our new configs too.

Reply:
@yctercero Yeah, good catch 👍